### DISCUSSION TITLE
Research topic: accounting for sampling and subsampling routines for aged fish

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/lingcod#198] [Ideas]
For the document, it would be good to know which age samples were subsampled for reads. For example, I have an email from @aliwhitman on 2020-08-14 "Oregon lingcod fin rays" that mentions both commercial and recreational age structures were subsampled to choose which fins to mount and read. This could affect marginal versus conditional ages. Please reply to this issue with any information you have.
- [ ] Find information for the following:
    * Commercial
        - [ ] WA
        - [ ] OR
        - [ ] CA
    *   Recreational
        - [ ] WA
        - [ ] OR
        - [ ] CA
- [ ] Determine if any sensitivity runs need to be conducted to explore marginal vs. CAAL b/c of how data were sampled or subsampled
- [ ] Add information to the document

**Comment by aliwhitman** [comment]
I have some information on the OR samples.  

**Commercial**
- no subsampling occurred for the new 2017 - 2019 ages, all samples were aged, with the exception of a few samples that were poorly mounted, so census! 
[ ] past years, I'm going to need to dig some more - it looks like there are more samples than were aged at a glance at PacFIN

**Recreational**
- this year, there are new ages from 2013 and 2017 - 2019 
- These were subsampled to 260 fins/year in proportion to the monthly proportion of annual landings 
- for years 2012 and 2014 - 2016 (the "new" ages for the last assessment), we were instructed to subsample to 260 fins per year but in a stratified random procedure 
- 130 fins per male/female, and then age more heavily at the tails of the length distributions 
- there were very specific procedures for the distribution of the subsampling for the last assessment that I have outlined in an email and I can provide that if need be
[ ] other years (<2012), I'm going to need to dig some more

**Comment by aliwhitman** [comment]
not sure why my check boxes aren't showing up :(

**Comment by kellijohnson-NOAA** [comment]
Tagged for the next assessment.

**Comment by brianlangseth-NOAA** [comment]
This is something that we should explore more

- [ ] Understand the scope of the issue
- [ ] Think about how to model this to address what we find in exploration

**Comment by chantelwetzel-noaa** [comment]
Here is the summary of this discussion from the 2017 STAR Panel report that determined this issue was most prevalent in age data from 2010 forward from Oregon and Washington, with the cause potentially due to multiple factors: 

> Histograms of aggregated age- and length-compositions of lingcod collected from the catches by
> Washington and Oregon commercial fishers suggested that the (marginal) age-compositions
> might not be representative of the corresponding length-compositions. However, the data for the
> histograms were aggregated over many years and could not be easily compared because of
> differing length-bin structures. Following discussion, it was agreed that pinpointing the source
> of the apparent discrepancies in the composition data would require a more detailed examination
> (described below). A suggestion that the commercial fishery age-composition data be excluded
> from the assessment model was rejected because both the STAR Panel and Assessment Team
> were reluctant to lose this potentially valuable information source. The STAR Panel endorsed
> further investigation to identify commercial fishery marginal age-composition data deemed to be
> unrepresentative.
> 
> Using information from the PacFIN Biological Data System for Washington and separately for
> Oregon, the STAT compiled annual numbers of fish-lengths and fish-lengths-with-ages based on
> 20-cm length-bins. The compilation provided convincing evidence that for 2010 and later years,
> the proportions of small fish in length samples from Washington that had been aged exceeded
> the proportions of larger fish that had been aged. The proportions of aged fish in the different
> length classes in the annual length samples from Oregon were highly variable, but again it was
> clear that samples for ages were not consistently representative of the length samples from which
> they had been drawn. If samples from either state were affected, the combined data for the
> northern stock would be biased. When compiling the marginal age-compositions the STAT had
> not adjusted for non-uniform sampling of fish for age-reading.
> 
> The STAR Panel concluded that the fishery marginal age-compositions from the northern stock
> were often not representative of the length-compositions, and thus the associated fishery
> marginal age-compositions for this stock are not representative of the stock’s age-composition.
> Subsequent discussion revealed issues sometimes associated with sampling lingcod. In
> Washington, permission to cut sampled fish (particularly larger fish) to extract spines for ageing
> and to sex those fish was often refused, thus resulting in biased samples. It is likely that port 
> samplers in Oregon and Washington encountered similar issues when obtaining lingcod samples
> for ageing. Furthermore, for the 2009 lingcod assessment as well as the new assessment ODFW
> staff had been instructed to deliberately over-sample small and large lingcod when selecting
> structures for age-reading. More detailed examination of the age- and length-composition is
> necessary to determine whether the bias extends to the age data for all years.
> 
> There was discussion of whether the available age data could be used as conditional age-atlength (CAAL) compositions. When constructing CAAL compositions using standard methods
> the data from individual hauls are aggregated without expanding for (possibly) different densities
> of fish at each sampling location. If there are age-dependent movements of lingcod, or the
> spatial distribution of the stock is age-dependent for some reason, then CAAL distributions based
> on sample data from surveys or fishery catches are also likely to be non-representative of the
> overall conditional distributions of age-at-length. In future research, consideration should be
> given to comparing CAAL distributions from different depths or regions to explore whether
> these are consistent with the hypothesis that CAAL data are representative of the overall CAAL
> distribution for the stock.
> 
> The STAR Panel advised that, for the current assessment, the CAAL compositions could be used
> but fishery marginal age-compositions should not be used.
